runner.sh

```diff
@@ -1,30 +1,40 @@
 #!/bin/bash
+
 set -e
 
 sudo touch requirements.txt
 user=nsls2
-image="condaforge"
+image="nsls2-analysis-2021-1.2"
 timestamp=$(date +%Y%m%d%H%M%S)
 env_name="nsls2-analysis-2021-1.2"
-#docker build . -t conda --target build
-#docker build . -t temp-image --target export
-#docker image build . -t $user/${image}:latest \
- #                  -t $user/${image}:${timestamp}
-#docker run -it --rm -v $PWD:/build $user/${image}:${timestamp} bash -l /build/export.sh
-#docker create -ti --name temp-container temp-image bash
-#docker cp temp-container:/build .
+
+if [ -e nsls2-analysis-2021-1.2.tar.gz ]
+then rm nsls2-analysis-2021-1.2.tar.gz
+     rm nsls2-analysis-2021-1.2.yml
+     rm nsls2-analysis-2021-1.2-sha256sum.txt
+fi
```

Comment:
```suggestion
if [ -f "${image}.tar.gz" ]; then
    rm -fv ${image}.tar.gz ${image}.yml ${image}-sha256sum.txt
fi
```
Or, alternatively, as discussed over the call, this can exit with the message to user such as:
```bash
if [ -f "${image}.tar.gz" ]; then
    echo -e "File ${image}.tar.gz exists. Move it or remove it before proceeding."
    exit 1
fi
```

Reply:
Changed.